README.md

```diff
@@ -1,192 +1,213 @@
-    * The uparrow in the upper left allows you to upload data
-    * if it is a large amount of data, consider creating a zip or tar archive first
-    * From within Jupyter Lab, you can also use the shell to 'scp' or 'git pull' from external sites that are publicly visible.
-    * What are our storage limits for uploaded data?
-    * How will I know if I run into a storage limit?
-  *  How to download data from the plaltform to my local machine?
-     * If it is a small file, you can right click on the file name in the file browser and scroll to `Download`
-     * if it is a large amount of data, consider creating a zip or tar archive first
-  * Home Directory
-    * When you log into the science console, the active directory is your $HOME directory.  This directory is unique to you: edits and uploads are not visible to other users.  
-        * Raen thinks there are directories in $HOME that are shared (e.g., `efs` and `s3`), and perhaps just mirrored or symlinked into $HOME. It would be nice to get clarification. Specifically:
-            * Which directories are/aren't shared?
-            * Which directories does the user have write access to, and are there any restrictions/considerations for putting stuff there (other than disk size)?
-            * Which directories can/should the user look in to discover data they have access to? (e.g., `s3` has various archive data, `efs` has some data shared by users, anything else?)
-  * Does my work persist between sessions?
-    * Files in your home directory will persist between sessions.
-    * pip installs will persist across kernel restarts, but not across logging out and back in.
-      * If you want software installs to be persistent, consider setting up an environment: See below under "Making a conda environment that persists across sessions"
-  * What is the info at the bottom of the jupyterlab window
-    * The github branch is listed as well as the name of the kernel in use
-    * the kernel is listed as either 'idle' or 'busy' which is useful to know if your kernel is working or has crashed.
-* How do I share large amounts of data from inside the SP with (international) collaborators?
+### What is a kernel and how do I choose one?
+  * In Jupyter, kernels are the background processes that execute cells and return results for display.
+  * To select the kernel on which you want to run your Notebook, go to the Kernel menu and choose Change Kernel. You can also click directly on the name of the active kernel to switch to another.
+    
+## Navigating jupyter lab
+### How do I start a new notebook?
+  * The blue `+` in the upper left brings you to the launcher where you can start a new, empty notebook or open a terminal window
+  * * ![new launcher](./new_launcher.png)
+### How do I get a terminal window?
+  * The blue `+` in the upper left brings you to the launcher where you can start a new notebook or open a terminal window
+  * ![terminal](./terminal.png)
```

Comment:
maybe merge these two, e.g. `### How to start a new notebook / get a new window`? the text is shared but the screenshots are not